source/modules/MEI.cmn.xml

```diff
@@ -1511,6 +1519,39 @@
       </rng:choice>
     </content>
   </elementSpec>
+  <elementSpec ident="damp" module="MEI.CMN">
+    <gloss versionDate="2023-05-27" xml:lang="en">damp</gloss>
+    <desc xml:lang="en">An instruction to dampen one or more strings of a plucked string instrument, such as the guitar or harp.</desc>
```

Comment:
On second thought, some composers have also used this symbol to indicate the damping of a percussion instrument. Maybe we state something like "An instruction to dampen a sound, usually…"?

Reply:
```suggestion
    <desc xml:lang="en">An instruction to dampen a sound, <abbr>e.g.</abbr>, one or more strings of a plucked string instrument, such as the guitar or harp.</desc>
```